### ISSUE TITLE
Deprecated rules?

### ISSUE THREAD

**Issue by pburkindine** [cartant/rxjs-tslint-rules#109]
Hello,

I am seeing indication in my lint feedback that two rules are deprecated:
```
rxjs-no-unbound-methods is deprecated. Rule is v6-only.
rxjs-prefer-observer is deprecated. Rule is v6-only.
```

... but do not see any indication of same in the changelog.  I am using latest (4.26.3) vs rxjs 6.5.3

Can I remove these rules?  I am depending on the functionality they provide.

**Comment by cartant** [comment]
The `rxjs-prefer-observer` deprecation seems to be a copy/paste error. That rule is not deprecated.

All of the rules that are specific to RxJS version 5 are deprecated. If you are using version 6 and `rxjs-compat`, the rules are still deprecated. IMO, `rxjs-compat` is not a long-term solution and I doubt it will be supported too far into the future.

**Comment by cartant** [comment]
Hmm. Those deprecation messages are intended to convey that those rules won't work entirely as intended for v5 codebases - or codebases that use v6 and `rxjs-compat` and the v5 dot-chaining style of composition. The rules look for `pipe` calls to find pipeable operators and there won't be any in v5.

I assume you are using v5.

I've not applied this deprecation consistently. I'll look at changing the message to make things more clear and will look at adding it to the metadata for other rules.

The intention is to say that the rule won't work the same in v5 and that supporting the rule in v5 is not going to happen.

**Comment by cartant** [comment]
This should be fixed in `4.27.1`.
